objax/module.py

```diff
@@ -115,47 +128,31 @@ def __init__(self,
             static_argnums: tuple of indexes of f's input arguments to treat as static (constants)).
                 A new graph is compiled for each different combination of values for such inputs.
         """
-        if vc is None:
-            if not isinstance(f, Module):
+        if not isinstance(f, Module):
```

Comment:
It's the same behavior as before. I didn't change it. It's already documented in "Understanding gradients...".

Reply:
Maybe add mentioning of this into docstring (or to RST files if docstring will be too long)?
I would expect that API docs is the primary read for users to understand what different arguments do, rather than in-depth tutorials.

Reply:
That would be better suited as a separate PR because this PR is not about changing the behavior of the API.

Reply:
It's fine to add in separate PR, I was thinking that it's small enough change of documentation which could be added right here.